dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/WorkflowItemConverter.java

```diff
@@ -88,6 +88,10 @@ public WorkflowItemRest fromModel(XmlWorkflowItem obj) {
         // info
 
         if (collection != null) {
+            // we set the status to true as we will discover validation error later in this block
+            // we could eventually leave the status to empty if we don't have collection information, this could be
+            // eventually the case when projection support will be included
+            witem.setStatus(true);
```

Comment:
Can you explain what this status boolean is?
It seems to be only used for workspace items and workflow items, but not documented.

Reply:
it was introduced to quickly signal the presence of errors in the data (missing information, unvalid values). This can always occurs when a configuration changes  and the previous data are not normalized. It is not really used as the submission/workflow functionalities will use the documented errors section with more details about the "issues". I'm going to remove it